Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -522,10 +548,8 @@ open class ChatMessageListVC: _ViewController,
         showThread(messageId: message.parentMessageId ?? message.id)
     }
 
-    open func messageContentViewDidTapOnQuotedMessage(_ indexPath: IndexPath?) {
-        log.info(
-            "Tapped a quoted message. To customize the behavior, override messageContentViewDidTapOnQuotedMessage."
-        )
+    open func messageContentViewDidTapOnQuotedMessage(_ quotedMessage: ChatMessage) {
```

Comment:
Breaking change as well. Does it make sense to break the API here?

Reply:
Yes for the reasons above